org.claws_mail.Claws-Mail.json

```diff
@@ -84,19 +84,19 @@
     },
     {
       "name": "liboauth",
-      "buildsystem": "simple",
+      "buildsystem": "autotools",
       "sources": [
         {
           "type": "archive",
           "url": "https://github.com/x42/liboauth/archive/186fb14843c1bc98b70ece02c74b5696b3de9ce7.zip",
           "sha256": "5d7a197cf56a7964efc37814b0121806972898e0cdf4fddab6e74249acf6fc83"
+        },
+        {
+          "type": "script",
```

Comment:
Does that actually work? You should probably add `"dest-filename": "autogen.sh"` too.